lib/rules/multiline-ternary.js

```diff
@@ -67,7 +72,15 @@ module.exports = {
                                 start: firstTokenOfTest.loc.start,
                                 end: lastTokenOfTest.loc.end
                             },
-                            messageId: "unexpectedTestCons"
+                            messageId: "unexpectedTestCons",
+                            fix: fixer => (hasComments ? null : (
+                                fixer.removeRange(
+                                    [
+                                        lastTokenOfTest.range[1],
+                                        questionToken.range[0]
+                                    ]
+                                )
+                            ))
```

Comment:
In some cases the range between `questionToken` and `firstTokenOfConsequent` should be removed:

```js
a ? 
b : c;
```

we could remove `[lastTokenOfTest.range[1], questionToken.range[0]]` if `lastTokenOfTest` and `questionToken` are not on the same line, and remove `[questionToken.range[1], firstTokenOfConsequent.range[0]]` if `questionToken` and `firstTokenOfConsequent` are not on the same line. That would sometimes remove only one range around `?`, sometimes both.

(same for `:`)